docs/sources/administration/service-accounts/about-service-accounts.md

```diff
@@ -0,0 +1,44 @@
+---
+title: About service accounts
+menuTitle: About service accounts
+description: 'This page contains detailed information about service accounts in Grafana'
+weight: 30
+---
+
+# About service accounts in Grafana
+
+A service account can be used to run automated or compute workloads. Applications use service account tokens to authorize themselves as a service account.
+
+> **Note:** Service accounts are available in Grafana 8.5+ as a beta feature, to enable service accounts refer to [Enable service accounts]({{< relref "./enable-service-accounts.md#">}}) section.
+
+A common use case for creating a service account is to perform operations on automated or triggered tasks. You can use service accounts to:
+
+- Schedule reports for specific dashboards to be delivered on a daily/weekly/monthly basis
+- Define alerts in your system to be used in Grafana
+- Set up an external authentication provider to manage users and permissions across an organization
+- Establish machine-to-machine communication
+- Interact with Grafana without logging in as a user
+
+You can also use service accounts in combination with fine-grained access control to grant users and teams specific scopes.
+
+You can associate a service account with multiple tokens. For example:
+
+- each team member can generate their own token
+-
```

Comment:
Not sure if this is something unintentional here? The formatting is broken and kind of starts and ends in a strange way.

Reply:
unintentional. also added some more examples